bigml_derive/src/updatable.rs

```diff
@@ -19,15 +19,9 @@ pub(crate) fn derive(ast: &DeriveInput) -> TokenStream {
 
         #[doc = #update_comment]
         #[derive(Clone, Debug, Default, PartialEq, Serialize)]
+        #[non_exhaustive]
         #vis struct #update_name {
             #( #update_fields )*
-
-            /// Placeholder to allow for future extension without breaking the
-            /// API. Pleae replace this with `#[non_exhaustive]` when it becomes
-            /// stable.
-            #[serde(skip)]
-            #[doc(hidden)]
-            #vis _placeholder: (),
```

Comment:
all these deletions are very satisfying

Reply:
Yup, `#[non_exhaustive]` finally stabilized a few days ago. I've been waiting for years. :-)